bytecomp/bytelink.ml

```diff
@@ -487,7 +487,7 @@ let link_bytecode_as_c ppf tolink outfile =
     Symtable.output_primitive_table outchan;
     (* The entry point *)
     output_string outchan "\
-\nvoid caml_startup(char ** argv)\
+\nvoid caml_startup(charnat ** argv)\
```

Comment:
Should probably be renamed to char_os once the other PR is merged, right?

Reply:
That's right, will update as necessary.

Reply:
In the interests of getting at least one of our CI systems back up and running, can we merge this before #1363?